src/filemenu/filemenu_info.c

```diff
@@ -48,24 +56,34 @@ void filemenu_info_draw_message_contents(
     switch (menu->page) {
         case 0:
             filemenu_draw_message(filemenu_get_menu_message(FILE_MESSAGE_FILE_26), baseX + 10, baseY + 4, 255, 0, 0);
-            draw_number(filemenu_menus[0]->selected + 1, baseX + 48, baseY + 6, DRAW_NUMBER_CHARSET_NORMAL, MSG_PAL_WHITE, 255, DRAW_NUMBER_STYLE_MONOSPACE | DRAW_NUMBER_STYLE_ALIGN_RIGHT);
+            draw_number(filemenu_menus[0]->selected + 1, baseX + 48, baseY + 6 + NUMBER_OFFSET_Y, DRAW_NUMBER_CHARSET_NORMAL, MSG_PAL_WHITE, 255, DRAW_NUMBER_STYLE_MONOSPACE | DRAW_NUMBER_STYLE_ALIGN_RIGHT);
             filemenu_draw_message(filemenu_get_menu_message(FILE_MESSAGE_HAS_BEEN_DELETED), baseX + 49, baseY + 4, 255, 0, 0);
             break;
         case 2:
+#if VERSION_CN
+            filemenu_draw_message(filemenu_get_menu_message(FILE_MESSAGE_COPY_FROM), baseX + 10, baseY + 7, 255, 0, 0);
+            filemenu_draw_message(filemenu_get_menu_message(FILE_MESSAGE_FILE_26), baseX + 42, baseY + 7, 255, 0, 0);
+            draw_number(filemenu_loadedFileIdx + 1, baseX + 84, baseY + 10, DRAW_NUMBER_CHARSET_NORMAL, MSG_PAL_WHITE, 255, DRAW_NUMBER_STYLE_MONOSPACE | DRAW_NUMBER_STYLE_ALIGN_RIGHT);
+            filemenu_draw_message(filemenu_get_menu_message(FILE_MESSAGE_TO), baseX + 84, baseY + 7, 255, 0, 0);
+            filemenu_draw_message(filemenu_get_menu_message(FILE_MESSAGE_FILE_26), baseX + 100, baseY + 7, 255, 0, 0);
+            draw_number(filemenu_iterFileIdx + 1, baseX + 140, baseY + 10, DRAW_NUMBER_CHARSET_NORMAL, MSG_PAL_WHITE, 255, DRAW_NUMBER_STYLE_MONOSPACE | DRAW_NUMBER_STYLE_ALIGN_RIGHT);
+            filemenu_draw_message(filemenu_get_menu_message(FILE_MESSAGE_PERIOD_34), baseX + 140, baseY + 7, 255, 0, 0);
+#else
```

Comment:
If would be nice if we could bring the differences out as defines instead of having a bunch of code basically copy/pasted here

Reply:
I considered/was trying that, but that would be 14 defines for one block. Times two languages (for now).

(positions are in `x, y` format)
|                               | US    | CN     |  
|----------|-------------|---------|
| "copy from"                   | 10,4  | 10,7   |  
| "file"                        | 84,4  | 42,7   |  
| int                           | 122,6 | 84,10  |  
| "to"                          | 10,18 | 84,7   |  
| "file"                        | 30,18 | 100,7  |  
| int                           | 68,20 | 140,10 |      
| "."                           | 65,18 | 140,10 |

Reply:
I think it'd still be visually much nicer, honestly. but it's annoying it's so spread out

Reply:
The reason I'm mostly hesitant is European languages; I don't know how their locale-specific code is done. If they have duplicate implementations with different orders, or just position phrases absolutely before others. If its the former, only having two languages use defines seems a bit silly.

Reply:
That's a good point. Idk, I'd still rather avoid chunks of virtually identical code copied like this, but I guess we can revisit it when we have the eu rom in the project